hr_employee_document/models/hr_employee_public.py

```diff
@@ -0,0 +1,54 @@
+# Copyright 2020 Tecnativa - Víctor Martínez
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class HrEmployeePublic(models.Model):
+    _inherit = "hr.employee.public"
+
+    document_ids = fields.One2many(
```

Comment:
This should be `Many2many` instead of `One2many` for avoiding side effects (in both places).

Reply:
Finally remove `document_ids` field because is unnecessary.